src/dev/src/app/quick-list/quick-list.demo.html

```diff
@@ -0,0 +1,31 @@
+<h2>Quick List</h2>
+<div class="clr-example">
+  <div class="main-container">
+    <div class="clr-row">
+      <div class="clr-col-1"></div>
+      <clr-quick-list class="clr-col-3"[allValues]="possibleActivityResults" [(values)]="selectedActivityResults"
```

Comment:
this clr-col-3 has no effect right now, but it should.
the selects should be as wide as this clr-col class describes

Reply:
I failed in making the select component cover all the reserved width, any hint please ?